tethys_cli/app_settings_commands.py

```diff
@@ -247,7 +315,10 @@ def get_setting_type(setting):
         SpatialDatasetServiceSetting: "ds_spatial",
         DatasetServiceSetting: "ds_dataset",
         WebProcessingServiceSetting: "wps",
+        CustomSettingBase: "custom_setting_base",
```

Comment:
I don't think we need `CustomSettingBase` in the `setting_type_dict`. Please remove.

Reply:
Done